trillian/ctfe/handlers.go

```diff
@@ -490,34 +503,16 @@ func addPreChain(ctx context.Context, li *logInfo, w http.ResponseWriter, r *htt
 	return addChainInternal(ctx, li, w, r, true)
 }
 
-// PingTreeHead retrieves a tree head for the given log, and updates the STH
```

Comment:
I was going to suggest moving this out of `handlers.go` as it's not really related.

Reply:
Yeah, I moved it to Trash. Does that look good?

Reply:
Couldn't be better.